components/drawer.css

```diff
@@ -0,0 +1,69 @@
+/* Drawer-Notification */
+
+.drawer-navigation {
+    background-color: var(--primary-200);
+    border-radius: 4px;
+    box-sizing: border-box;
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+    gap: var(--size-14);
+    height: 100%;
+    max-width: 12rem;
+    padding: var(--size-12);
+    overflow-y: auto;
+}
+
+.title-for-navigation {
```

Comment:
Have Small class names which can convey the meaning like .nav-title for this and .drawer-nav for above etc.

Reply:
Yeah right! I will look into it after merging. Still a little hazy about naming stuff. I will definitely work on this.